scss/_book-list.scss

```diff
@@ -0,0 +1,39 @@
+.book {
+  @include card-shadow;
+  display: flex;
+  flex-direction: column;
+  max-width: 20%;
+  padding: 32px 32px 20px 32px;
+
+  &-list {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 40px 0;
+    max-width: $desktop-width;
+  }
+
+  &__cover {
+    margin-bottom: 10px;
+    max-width: 100%;
+  }
+
+  &__info {
+    display: flex;
+    flex-direction: column;
+
+    &__title {
+      @extend %title;
+      font-size: 20px;
+      line-height: 2;
+    }
+
+    &__author {
+      @extend %content;
+    }
+  }
+}
+
+.m-right-bottom-40 {
```

Comment:
I think there's something cleaner for this kind of issue. Check `nth-child` 😉

Reply:
Done with css > selector 😄